include/ttmlir/Target/Utils/MLIRToFlatbuffer.h

```diff
@@ -478,7 +478,11 @@ toDebugInfo(::flatbuffers::FlatBufferBuilder &fbb, std::string const &name,
             ModuleOp module) {
   std::string source;
   llvm::raw_string_ostream os(source);
-  module->print(os);
+
+  mlir::OpPrintingFlags flags;
+  flags.enableDebugInfo(); // Enable the loc dumping
+  module->print(os, flags);
+
```

Comment:
Printing the MLIR module with debug info into the flatbuffer. I'm not sure if this is the behavior we want, but it could be useful to show the locations that are being propagated in the results.